get-started/mcp/sdk-integrations/openai-agents.md

```diff
@@ -0,0 +1,326 @@
+
+!!! info "Self-hosted option"
+    For self-hosted MCP, use `http://localhost:3001/stream` and your kluster.ai API key.
+
+## Enable MCP
+
+If you prefer, you can enable MCP and obtain your token via API calls instead of using the platform interface. This approach allows you to automate the token retrieval process and integrate it directly into your application setup workflow.
+
+```python
+import requests
+
+# Enable MCP
+response = requests.post(
+    "https://api.kluster.ai/v1/mcp/enable",
+    headers={"Authorization": f"Bearer {api_key}"}
+)
+
+# Get status with token
+status = requests.get(
+    "https://api.kluster.ai/v1/mcp/status", 
+    headers={"Authorization": f"Bearer {api_key}"}
+)
+mcp_token = status.json()["apiKey"]
+```
+
+## Build an interactive chatbot
+
+This example creates an interactive chatbot where you can ask questions and see kluster Verify validate the responses in real-time, demonstrating how verification enhances conversational AI.
+
+```python
+from agents import Agent, Runner, set_tracing_disabled, set_default_openai_api, OpenAIChatCompletionsModel
+from agents.mcp.server import MCPServerStreamableHttp
+from openai import AsyncOpenAI
+import asyncio
+import getpass
+
+# Configure SDK
+set_tracing_disabled(True)
+set_default_openai_api("chat_completions")
+
+# Get credentials
+api_key = getpass.getpass("Enter your kluster.ai API key: ")
+mcp_token = getpass.getpass("Enter your MCP token: ")
+
+# Create kluster.ai client
+kluster_client = AsyncOpenAI(
+    base_url="https://api.kluster.ai/v1",
+    api_key=api_key
+)
+# Create MCP server
+mcp_server = MCPServerStreamableHttp(
+    params={
+        "url": "https://api.kluster.ai/v1/mcp",
+        "headers": {"Authorization": f"Bearer {mcp_token}"},
+        "timeout": 15,
+        "sse_read_timeout": 15
+    },
+    client_session_timeout_seconds=30  # Increase from default 5 seconds to 30 seconds
+)
+
+# Create chatbot agent
+agent = Agent(
+    name="VerifyChatbot",
+    instructions="Use kluster Verify to validate factual claims and provide reliable responses.",
+    model=OpenAIChatCompletionsModel(
+        model="deepseek-ai/DeepSeek-V3-0324",
+        openai_client=kluster_client
+    ),
+    mcp_servers=[mcp_server]
+)
+
+# Interactive chat loop
+async def main():
+    await mcp_server.connect()
+    
+    print("🤖 Chatbot ready! Type 'quit' to exit.")
+    
+    while True:
+        user_input = input("\n👤 You: ").strip()
+        
+        if user_input.lower() in ['quit', 'exit', 'q']:
+            print("👋 Goodbye!")
+            break
+            
+        if user_input:
+            result = await Runner.run(agent, user_input)
+            print(f"🤖 Bot: {result.final_output}")
+
+asyncio.run(main())
+```
+
+## Complete example
```

Comment:
A section is overkill here. You can just add this to the end of the above section